### DISCUSSION TITLE
How to avoid an extra dlv.exe open in another terminal?

### DISCUSSION THREAD

**Discussion by longguzzz** [mfussenegger/nvim-dap#698] [Q&A] [answered]
After using `lua require'dap'.continue()` and choose `debug` to run, another terminal with title "C:\Users\<username>\go\bin\dlv.EXE" will prompt up. And the focus transfers to that one. But I already have one in origianl terminal.

It seems that the default situation for others is opening window in the same terminal without an extra terminal. And the other question focus on how to open an extra termnial.

 I don't want to change focus. Is there any method to avoid this extra terminal? Or whether dlv.EXE can run silently?

These are what this two termnial like:

1. The terminal using vim.
![image](https://user-images.githubusercontent.com/96779663/192147940-2a91be0a-1c0c-4594-bfd4-02e6330268b0.png)

2. the second terminal prompts up with non contents throught the debug process.
![image](https://user-images.githubusercontent.com/96779663/192148026-35163cec-0828-4586-9c45-1f21eecfe999.png)

---- update ----

 Changing the all four `dap.defaults.fallback` options in dap.lua has no effects. So, I think it may be more relevant to my vim configuration? I am using lunarvim, so I also open a question there. 
[Using dlv for go debugging will prompt up a second cmd, which steals focus. #3078](https://github.com/LunarVim/LunarVim/discussions/3078)

**Comment by mfussenegger** [comment, marked as answer]
Can you try setting `detached` to `false` in your adapter configuration? See `:h dap-adapter` and look for the `executable` settings.

**Reply by longguzzz** [reply]
Yes, it works! Thank you!
